process-compose/movement-full-node/process-compose.test-migrate-biarritz-rc1-to-pre-l1-merge.yml

```
@@ -0,0 +1,18 @@
+version: "3"
+
+processes:
+
+  setup:
+    environment:
+      - "KNOWN_FRAMEWORK_RELEASE=biarritz-rc1"
+      - APTOS_ACCOUNT_WHITELIST=$DOT_MOVEMENT_PATH/default_signer_address_whitelist
+      - MAPTOS_PRIVATE_KEY=random
+
+  test-migrate-elsa-to-biarritz-rc1:
```

Shouldn't this be more like 
```suggestion
  test-migrate-biarritz-rc1-to-pre-l1-merge:
```